util/generateRandomSub.js

```diff
@@ -0,0 +1,5 @@
+const crypto = require('crypto');
+
+const generateRandomSub = () => crypto.randomBytes(12).toString('hex');
```

Comment:
these were decimals in the previous seeds, just double-checking that hex chars are valid here?

Reply:
nvm, I see the test where we used random strings, so guessing this is totally fine